packages/behandling-frisinn/src/panelDefinisjoner/faktaFrisinnPanelDefinisjoner.tsx

```diff
@@ -36,6 +37,23 @@ const faktaPanelDefinisjoner: FaktaPanelDefinisjon[] = [
     showComponent: () => true,
     getData: () => ({}),
   },
+  {
+    urlCode: faktaPanelCodes.BEREGNING,
+    textCode: 'BeregningInfoPanel.Title',
+    aksjonspunkterCodes: [
+      aksjonspunktCodes.VURDER_FAKTA_FOR_ATFL_SN,
```

Comment:
Skal alle desse aksjonspunktene vere tilgjengelig for FRISINN?

Reply:
Endret disse og fjerner overstyringsmulighet